habitat_sim/agent/controls/default_controls.py

```diff
@@ -137,3 +137,19 @@ def __call__(self, scene_node: SceneNode, actuation_spec: ActuationSpec):
         _rotate_local(
             scene_node, -actuation_spec.amount, _X_AXIS, actuation_spec.constraint
         )
+        
+        
+@registry.register_move_fn(body_action=False)
+class LookClock(SceneNodeControl):
+    def __call__(self, scene_node: SceneNode, actuation_spec: ActuationSpec):
+        _rotate_local(
+            scene_node, actuation_spec.amount, _Z_AXIS, actuation_spec.constraint
+        )
+
+
+@registry.register_move_fn(body_action=False)
+class LookAnti(SceneNodeControl):
```

Comment:
```suggestion
class RotateSensorAntiClockwise(SceneNodeControl):
```

Reply:
Why is this AntiClockwise instead of CounterClockwise?